### DISCUSSION TITLE
Is it supported: descriptions on queries/mutations/subscriptions?

### DISCUSSION THREAD

**Discussion by stevenschmatz** [graphql-hive/graphql-eslint#907] [Q&A] [answered]
I see `require-descriptions` rule, but it looks like this only applies to types. Would love to enforce descriptions on queries, mutations, and subscriptions.

**Comment by dimaMachina** [comment]
I think you're right, I added a new option `OperationDefinition` for this rule. Could you try `3.7.0-alpha-a9b0ecf.0` alpha version?

**Comment by dotansimha** [comment, marked as answer]
Fixed in `@graphql-eslint/eslint-plugin@3.8.0` 🎉
